Assets/Scripts/HandControl.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HandControl : MonoBehaviour
+{
+	public float speed = 10.0f;
+	public float turnSpeed = 360.0f;
+
+	// Use this for initialization
+	void Start () {
+		
+	}
+	
+	// Update is called once per frame
+	void Update () 
+	{
+
+		var scrollVal = Input.GetAxis("Mouse ScrollWheel");
+	
+		// Scroll down to lower hand
+		if (scrollVal < 0f)
+		{
+			Vector3 position = this.transform.position;
+			position.y -= speed*Time.deltaTime;
+			this.transform.position = position;
```

Comment:
Like we talked about before, you can replace transform.position assignments with method calls to rigidbody.AddForce();
Also, since getting the transform or the rigidbody requires a call to GetComponent() every time, it's good practice to cache that value in a variable (you can do this in Start() with code like "myRigidbody = GetComponent<Rigidbody>();"